OpenRA.Mods.Common/LoadScreens/LogoStripeLoadScreen.cs

```diff
@@ -79,12 +79,15 @@ public override void Display()
 			r.EndFrame(new NullInputHandler());
 		}
 
-		public override void Dispose()
+		protected override void Dispose(bool disposing)
 		{
-			if (sheet != null)
-				sheet.Dispose();
+			if (disposing)
```

Comment:
Well there is no `a` vs `b` to worry about here, and this is inconsistent with the rest of our formatting.